src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -154,6 +184,13 @@ class CgroupProcessingError(ProcessingError):
     pass
 
 
+class JobValueError(Exception):
+    """
+    Errors in PBS job resource values.
+    """
+    pass
+
+
```

Comment:
No reason; that is a typographic accident. The important change is that it is derived from Exception and not UserError, which has nefarious side-effects on user jobs when the scheduler gets things wrong wrt resources that are available on vnodes (if derived from UserError, the original derivation will make the user job be _deleted_ instead of requeued, even though the job may well be valid on nodes for which scheduler and cgroup hook agree on the available resources).

I will move it back to its original location.

Reply:
On the other hand, moving it back to its original location would make it easier to see just what changed (instead of giving the impression the section only moved), so I'll move it back.